imap/http_caldav_sched.h

```diff
@@ -173,7 +173,8 @@ extern void sched_request(const char *onuserid, const strarray_t *schedule_addre
                           icalcomponent *oldical, icalcomponent *newical);
 extern void sched_reply(const char *onuserid, const strarray_t *schedule_addresses,
```

Comment:
Nit: the `onuserid` argument name got renamed to `userid` in the function implementation.